### ISSUE TITLE
calculating execution time

### ISSUE THREAD

**Issue by ChulhoonJang** [MarvinTeichmann/KittiSeg#93]
Thank you for your codes. It is very helpful to understand and implement FCN.

I am just curious about the code regarding execution time calculation.

As I understand, dt is second. Thus, I agree that the execution time and fps are calculated as 152 and 155 lines. However, in KittiSeg/evals/kitti_eval.py, 141 line, you divided dt with 10. Could you tell me why?

**Comment by bendidi** [comment]
division by 10 is because we run the FCN in a loop over 10 image so to get time per image you divide by 10

**Comment by ChulhoonJang** [comment]
Is the aim to obtain an average time for inference?

**Comment by bendidi** [comment]
yes the goal is to get the "time by frame" or average inference time as you said it , and if you inverse it you will get "frame per time"=fps

**Comment by ChulhoonJang** [comment]
Thank you. @bendidi !! I got it!
